.eslintrc.json

```diff
@@ -151,8 +150,20 @@
       "extends": ["plugin:@angular-eslint/template/recommended"],
       "rules": {}
     },
+    // associated components eslint rules to match DXCB
     {
-      "files": ["**/custom-constellation/**"],
+      "files": ["**/custom-constellation/**/*.@(js|jsx)"],
+      "rules": {
+        "import/no-relative-packages": "error",
```

Comment:
Shouldn't following be set to `off` rather than `error` since they're specific to Constellation:
1. import/no-relative-packages
2. @typescript-eslint/no-shadow
3. no-undef
4. no-console

no-console is something we can set to `error` but would need inline disable statements to make those work.